eclcli/bare/bareclient/v2/servers.py

```diff
@@ -740,11 +740,14 @@ def clear_password(self, server):
         return self._delete("/servers/%s/os-server-password"
                             % base.getid(server))
 
-    def stop(self, server):
+    def stop(self, server, body):
```

Comment:
'body' is abstract. Can you rename it to 'type'?
```suggestion
    def stop(self, server, type):
```